spec/models/review_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'rails_helper'
+
+RSpec.describe Review, type: :model do
+  describe '#validations' do
+    it { is_expected.to validate_presence_of :rating }
+    it { is_expected.to validate_presence_of :reviewer_name }
+    it { is_expected.to validate_numericality_of(:rating)
```

Comment:
FYI in ruby,
when writing multiline blocks we prefer to use `do |x| ..end`.

if the block is the same line then we would normally use `{ |x| .. }`